entrypoint/main.sh

```diff
@@ -60,7 +63,7 @@ su ${APP_USER} --command "ampinstmgr ShowInstanceInfo Main | grep \"Start on Boo
 
 # Startup
 echo "Starting AMP..."
-su ${APP_USER} --command "ampinstmgr StartBoot"
+su ${APP_USER} --command "ampinstmgr StartBoot" &
```

Comment:
It might be a nieche-case, but i experienced problems in the past when amp was in foreground and was not able to start correcrly ehich resulted in the whole container to crash. The best workaround i found was starting it in background. But that was some time ago and it might have changed in the mean time, so i think we could leave it in the forefround.

Reply:
Ok, let's leave it as foreground and let me know if the issue pops up again!